bom/runtime/pom.xml

```diff
@@ -660,6 +660,11 @@
                 <artifactId>quarkus-spring-boot-properties</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-spring-config-server-client</artifactId>
```

Comment:
Should we use `server` and `client` in the same artifact ID? :smiley:

Reply:
`quarkus-spring-cloud-config-client` may be better I think

Reply:
TBH the name has been bugging me as well...  Having a both the `server` and `client` indeed sucks :).

Yeah, let's go with your proposal, it's way better than the current name